sparse/tests/test_array_function.py

```diff
@@ -0,0 +1,22 @@
+import sparse
+from sparse.utils import assert_eq
+import pytest
+np = pytest.importorskip('numpy', minversion='1.16')
```

Comment:
Also add a skip here based on the environment variable. We don't want tests to fail if one didn't export the var. :slightly_smiling_face: